### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by lrennels**

### CODE REVIEW

**Comment by lrennels** [comment]
@arnavgautam if you get the chance to look at this let me know, I think this is a good incremental improvement to the docs with some `[TODO]` placeholders that I can fill over time ... but merging this as is after approval would be good to start.

File: docs/src/explanations/exp_pkgs.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Explanations: Models as Packages
+
+## Models as Packages
+
+[Pkg](https://docs.julialang.org/en/v1/stdlib/Pkg/index.html) (more detail [here](https://julialang.github.io/Pkg.jl/v1/) is Julia's builtin package manager, and handles operations such as installing, updating and removing packages. It is often convenient to set Mimi models up as julia packages using the steps described [here](https://julialang.github.io/Pkg.jl/v1/creating-packages/). "A package is a project with a `name`, `uuid` and version entry in the `Project.toml` file, and a `src/PackageName.jl` file that defines the module `PackageName`. This file is executed when the package is loaded."
```
Comment (arnavgautam) [inline docs/src/explanations/exp_pkgs.md:5]: Need a second ")" after "more detail here"?

File: docs/src/explanations/exp_pkgs.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Models can be registered or unregistered, as described in the next section, but it is **not necessary** to register a package in order to use the `Pkg` interface.
+
+### Example
+
+The [MimiDICE2016](https://github.com/AlexandrePavlov/MimiDICE2016.jl) model is an unregistered package. As it's README instructs, it can be accessed by:
```
Comment (arnavgautam) [inline docs/src/explanations/exp_pkgs.md:11]: nit: "its"

**Review by arnavgautam — DISMISSED** [review]
Other than minor possible typos, LGTM

**Review by arnavgautam — APPROVED** [review]

**Pull request merged**